module/ldbc-connector/shared/src/main/scala/ldbc/connector/net/packet/request/AuthSwitchResponsePacket.scala

```diff
@@ -25,7 +23,7 @@ import cats.syntax.all.*
  *   Description: authentication response data
  */
 case class AuthSwitchResponsePacket(
```

Comment:
If you seriously care about (binary) compatibility,  I recommend using the following pattern to hide some `case class` drawbacks.

```scala
trait AuthSwitchResponsePacket:
  def hashedPassword: ByteVector

object AuthSwitchResponsePacket:
  def apply(hashedPassword: ByteVector) = new Impl(hashedPassword)
  private case class Impl(hashedPassword: ByteVector) extends AuthSwitchResponsePacket:
     // todo